nxp_s32k/libuavcan/README.md

```diff
@@ -1,5 +1,5 @@
 # S32K Libuavcan V1
-### Bare-metal media layer driver for the NXP S32K14x family of automotive-grade microcontrollers, featuring CAN-FD running at 4 Mb/s and 1 Mb/s in data and nominal phases, respectively.
+### Bare-metal media layer header-only driver for the NXP S32K14x family of automotive-grade microcontrollers, featuring CAN-FD running at 4 Mb/s and 1 Mb/s in data and nominal phases, respectively.
```

Comment:
Is it worth mentioning more detail as to *why* this driver was created?

Reply:
What type of aspects should I consider?